include/rootston/desktop.h

```diff
@@ -52,19 +61,24 @@ struct roots_desktop {
 struct roots_server;
 
 struct roots_desktop *desktop_create(struct roots_server *server,
-		struct roots_config *config);
+	struct roots_config *config);
 void desktop_destroy(struct roots_desktop *desktop);
 
 void view_destroy(struct roots_view *view);
 struct roots_view *view_at(struct roots_desktop *desktop, double lx, double ly,
-		struct wlr_surface **surface, double *sx, double *sy);
+	struct wlr_surface **surface, double *sx, double *sy);
 void view_activate(struct roots_view *view, bool activate);
 
+bool layer_surface_is_at(struct roots_desktop *desktop,
+	struct wlr_layer_surface *layer_surface, double lx, double ly,
+	double *sx, double *sy);
```

Comment:
This is a layer_surface centric function and has prefix `layer_surface` but has desktop as the first parameter, only uses the layout of the desktop, and lives in desktop.h.

Reply:
And also for consistency, should be `layer_surface_at` as in other parts of the code that give something from a position.

Reply:
For the consistency point, I don't agree: `view_at` returns the view which is at a position, and this function does something different: it tells whether or not a surface layer is at a position. Hence the different names.

Reply:
That's true.